docs/building_the_docs.md

```diff
@@ -0,0 +1,7 @@
+(building_the_docs)=
+# Building the docs
+
+From the directory ``docs``, first install the python dependencies with ``pip install -r requirements-doc.txt``. 
+Then to build the docs run ``make html``.
+The built documentation will be in the subdirectory ``docs/_build/html``. To view them, cd into ``_build/html`` and start a file server, 
+for instance ``http-server``.
```

Comment:
Thanks @hoodmane ! I think we should add this to the contributing docs https://github.com/iodide-project/pyodide/blob/master/docs/contributing.md#documentation there is already a section on how to contribute to the documentation there.

Reply:
Okay I will move it there.